server/Dockerfile

```diff
@@ -47,82 +50,62 @@ FROM "${DOCKER_HUB_PROXY}debian:bullseye-slim" as php-build
         ca-certificates \
         php \
         php-dev \
+        php-xml \
         php-pear \
         librdkafka-dev \
         libsimdjson-dev \
         git \
         && apt-get autoremove -y && apt-get clean -y && rm -rf /var/lib/apt/lists/*
-
-    RUN pecl channel-update pecl.php.net
```

Comment:
Any reason why we don't need to do this?

Reply:
Re-added